### ISSUE TITLE
Error running migrations

### ISSUE THREAD

**Issue by chrisedington** [joshmn/caffeinate#4]
Hi,

Running into an issue running the migrations on Rails 5.0.4.

Ran the generator, which showed a line I haven't seen before, wasn't sure if it had consequences:

```
Running via Spring preloader in process 19482
      create  config/initializers/caffeinate.rb
      create  app/drippers/application_dripper.rb
**File unchanged! The supplied flag value not found!  config/routes.rb**
Copied migration 20210105060001_create_caffeinate_campaigns.caffeinate.rb from caffeinate
Copied migration 20210105060002_create_caffeinate_campaign_subscriptions.caffeinate.rb from caffeinate
Copied migration 20210105060003_create_caffeinate_mailings.caffeinate.rb from caffeinate
```

Then I ran a rake db:migrate and got the following errors:

```
rails aborted!
ArgumentError: Unknown migration version "6.0"; expected one of "4.2", "5.0"
```

Seems the migrations set to Rails 6 only?

**Comment by joshmn** [comment]
hey Chris! Sorry for the delayed reply. I didn't see this in my notifications.

I haven't added a version check yet for migrations. Changing the versions on the migrations should work. I'll get a PR up shortly to address this.

**Comment by joshmn** [comment]
Should be fixed in 0.14 :)
